rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -628,7 +672,7 @@ class AWSBakeHandlerSpec extends Specification {
         aws_target_ami: targetImageName,
         repository: DEBIAN_REPOSITORY,
         package_type: BakeRequest.PackageType.DEB.packageType,
-        packages: fullyQualifiedPackageName,
+        packages: unqualifiedPackageName,
```

Comment:
Oh wow. Thanks for pointing this out. I just read up on the spock framework and the right shift operator. I've misunderstood this: ``` 1 * imageNameFactoryMock.buildPackagesParameter(osPackages) >> unqualifiedPackageName ``` to mean ``` verify that this method should be called once and that it returns unqualifiedPackageName ``` when the return actually tells the mock what it should return. Sorry about that, I'll take a fresh look at the tests with this in mind.